tests/refer.spec.ts

```diff
@@ -57,6 +57,42 @@ const showsPersonalDetails = async (page: Page): Promise<void> => {
   await page.getByRole('link', { name: 'Back', exact: true }).click()
 }
 
+const entersProgrammeHistory = async (page: Page): Promise<void> => {
+  await page.getByRole('link', { name: 'Review Accredited Programme history' }).click()
+
+  await expect(page.locator('h1')).toHaveText('Accredited Programme history')
+  await page.getByTestId('add-history-button').click()
+
+  await expect(page.locator('h1')).toHaveText('Add Accredited Programme history')
+  await page.getByLabel('Horizon').click()
+  await page.getByRole('button', { name: 'Continue' }).click()
+
+  await expect(page.locator('h1')).toHaveText('Add Accredited Programme details')
+  await page.getByTestId('custody-setting-option').click()
+  await page.getByLabel('Enter the prison (if known)').fill('Stocken (HMP)')
+  await page.getByTestId('complete-outcome-option').click()
+  await page.getByLabel('Enter the year completed (if known)').fill('2020')
+  await page.getByLabel('Provide additional detail (if known)').fill('Spiffing')
+  await page.getByLabel('Provide the source').fill('The person sat next to me')
+  await page.getByRole('button', { name: 'Continue' }).click()
+
+  await expect(page.locator('h1')).toHaveText('Accredited Programme history')
+  await expect(page.locator('.moj-banner__message')).toContainText('You have successfully added a programme.')
+  const summaryCard = page.locator('.govuk-summary-card').last()
+  await expect(summaryCard.locator('.govuk-summary-card__title')).toContainText('Horizon')
+  await summaryCard.locator('.govuk-summary-card__action').last().click()
+  // await summaryCard.getByRole('link', { name: 'Remove', exact: true }).click()
```

Comment:
Is this supposed to replace line 83 at some point?

Reply:
https://github.com/ministryofjustice/hmpps-accredited-programmes-e2e/pull/39/commits/079969347f310d7355c2e5c8f7939ee02d904aac#r1356894388